data/campaigns/Heir_To_The_Throne/scenarios/03_Blackwater_Port.cfg

```diff
@@ -0,0 +1,679 @@
+                message=_"Uncle Kaylan— <b>Sir</b> Kaylan fought bravely... but in the end he fell, piereced by Asheviere’s spears. It is thanks to his sacrifice that Blackwater Port still stands free."
+            [/message]
+
+            {IF_VAR s3_early_win equals yes (
+                [then]
+                    [message]
+                        speaker=Simyr
+                        message= _ "Alas, this is ill news indeed. A hero’s death, but a death nonetheless. Kaylan had hoped to send some of our number with you on your journey, but with his passing we cannot afford to send the Horsemen and Cavalrymen he had hoped to."
+                    [/message]
+                    [message]
+                        speaker=Haldiel
+                        message= _ "Yet with Asheveire’s armies defeated today, surely we can spare at least a few of our number? Though it is true that the bulk of our forces must remain behind to guard the port, we can still partially fufill Sir Kaylan’s last wish, Sir Simyr."
+                    [/message]
+                    [message]
+                        speaker=Simyr
+                        message= _ "Very well. Haldiel and I shall join you on your quest, Prince Konrad."
+                    [/message]
+                    {MODIFY_UNIT id=Haldiel side 1}
+                    {MOVE_UNIT id=Haldiel $s3_konrad_position.x $s3_konrad_position.y}
+                    {MODIFY_UNIT id=Simyr side 1}
+                    {MOVE_UNIT id=Simyr $s3_konrad_position.x $s3_konrad_position.y}
+                [/then]
+                [else]
+                    [message]
+                        speaker=Simyr
+                        message= _ "Alas, this is ill news indeed. A hero’s death, but a death nonetheless. Kaylan had hoped to send some of our number with you on your journey, but with his passing we can no longer afford to. Konrad, you will have to journey on from here with only Peasants and Woodsmen by your side."
+                    [/message]
+                [/else]
+            )}
+        [/then]
+        [else]
+            [message]
+                speaker=Kaylan
+                message= _ "A glorious victory — thank you for your help, compatriots! I must remain here to safeguard the port, but with false queen’s troops repelled many of my riders are free to accompany you on your quest. Konrad, may they serve you well!"
+            [/message]
+            {NEW_RECRUIT2 (Cavalryman,Horseman) _"Konrad can now recruit Cavalrymen and Horsemen!" human-loyalists/cavalryman/cavalryman.png human-loyalists/horseman/horseman.png}
+            {IF_VAR s3_early_win equals yes (
+                [then]
+                    [message]
+                        speaker=Kaylan
+                        message= _ "...And with the invading force so thoroughly defeated, I can afford to give you more riders than I had originally planned. I place two of my city’s finest horsemen in your service."
```

Comment:
Should horsemen be capitalised here (since unit types are capitalised in other lines here).